nativescript-slides.ts

```diff
@@ -9,6 +9,8 @@ import * as AnimationModule from 'ui/animation';
 import * as gestures from 'ui/gestures';
 import {AnimationCurve} from 'ui/enums';
 
+const LayoutParams = android.view.WindowManager.LayoutParams;
+
```

Comment:
my android d.ts file doesn't have layoutParams, did you get yours from tns-platform-declarations... I think i'm behind.